domain/src/main/java/life/qbic/apps/datamanager/notifications/MessageParameters.java

```diff
@@ -0,0 +1,33 @@
+package life.qbic.apps.datamanager.notifications;
+
+import java.time.Instant;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+public final class MessageParameters {
+
+  public final String messageType;
```

Comment:
why is this a string and not an enum?

Reply:
ah, because this is taken from class names?

Reply:
exactly